inventory/splunk_defaults_windows.yml

```diff
@@ -46,7 +46,9 @@ splunk:
         ssl: False
         cert:
         password:
-        ca: 
+        ca:
+    appserver_port: 8065
```

Comment:
Nitpicky, but I regret making some of the options here flattened (ex. `http_port`, `http_enableSSL`, etc.). Since these are new additions, I would much prefer to make these nested groups like:
```
splunk:
    kvstore:
        port: 8191
```
to support future maintainability of tuning parameters in specific components of Splunk.

Reply:
fixed for `kvstore` and `appstore`